app/views/main.scala.html

```diff
@@ -50,9 +50,9 @@
                         -->
                             <!-- Navbar buttons -->
                         <ul class="nav nav-pills">
-                            <li role="presentation"><a id="#" title="Top" onclick="topFunction();return false;">Top</a></li>
-                            <li role="presentation"><a id="#" title="Trending" onclick="trendingFunction();return false;">Trending</a></li>
-                            <li role="presentation"><a id="#" title="Hashtag" onclick="hashtagFunction('testHashtag');return false;">Hashtag</a></li>
+                            <li role="presentation"><a id="#" title="Top" onclick="topFunction('0');return false;">Top</a></li>
+                            <li role="presentation"><a id="#" title="Trending" onclick="trendingFunction('0');return false;">Trending</a></li>
+                            <li role="presentation"><a id="#" title="Hashtag" onclick="hashtagFunction('testing', '0');return false;">Hashtag</a></li>
```

Comment:
We don't have the search bar implemented in this version, so there's no way for users to pass in which hashtag they want to look for, so making it prod ready will have to wait until Improve UI gets merged into master, and this one gets merged in, then we can make a new branch for implementing the hashtag front-end